Devscord.DiscordFramework/Framework/Server.cs

```diff
@@ -242,5 +245,11 @@ public static async Task<IEnumerable<string>> GetExistingInviteLinks(ulong serve
             var invites = await guild.GetInvitesAsync();
             return invites.Select(x => x.Url);
         }
+
+        private static Task BotDisconnected(Exception exception)
+        {
+            Log.Warning($"Bot disconnected {exception.Message} stack trace: {exception.StackTrace}");
```

Comment:
jak używasz seriloga to nigdy nie używaj zapisu $"aa {zmienna} aa"
używaj Log.CośTam("aa {zmienna} aa", zmienna)
ma to 2 plusy

pierwszy jest taki, że treść zmiennej jest w innym kolorze w konsoli i lepiej widać

a drugi - teraz nie używamy, ale dobrze byłoby to zmienić (wcześniej używaliśmy, ale było problematyczne przez wydajność)
czyli serilog pozwala na logi strukturyzowane - w jsonie
gdzie mamy informacje o requeście itp
ale nie tylko, bo poza tym mamy informacie o zmiennych w wiadomości jako json
przez co dużo szybciej można wyszukiwać treści w takich danych
i bardzo łatwo można parsować logi, gdybyśmy kiedyś tego potrzebowali
lepiej tego nie robić, ale gdybyśmy musieli to byłoby dużo prościej

tym bardziej że to tylko drobna zmiana w zapisie stringa